src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerClientViewModel.cs

```diff
@@ -482,6 +477,12 @@ public List<PackageManagerSearchElement> ListAll()
         {
             CachedPackageList = new List<PackageManagerSearchElement>();
 
+            // Attempt to load user votes prior to using it
+            if (AuthenticationManager.LoginState.Equals(LoginState.LoggedIn) && Uservotes == null)
```

Comment:
So you are basically caching UserVotes.
Will it be a problem that UserVotes will not refresh for subsequent calls to `public List<PackageManagerSearchElement> ListAll()` ?

Reply:
There is a 15 minute lag until the database picks up the changes, as far as I know. The data is specific to the user. Which means, that they won't be able to see the subsequent change until the database picks it up, again this is as far as my knowledge goes on how fast the data will be registered. 

With that, I think there is no problem that the UserVotes won't refresh in subsequent calls of ListAll().